calicoctl/commands/node/checksystem.go

```diff
@@ -133,6 +138,12 @@ func checkKernelModules() error {
 	// File path to Builtin kernel modules
 	modulesBuiltinPath := fmt.Sprintf("/lib/modules/%s/modules.builtin", kernelVersionStr)
 
+	// File path to module configs in boot time
+	modulesBootPath := findBootFile(kernelVersionStr)
+
+	// File path for loaded iptables modules
+	modulesLoadedIPtables := "/proc/net/ip_tables_matches"
```

Comment:
I rather use another method but this was the only way that I could've think of iptables modules `"xt_icmp","xt_icmp6"`.